distributions/otelcol-ebpf-profiler/.goreleaser.yaml

```
@@ -0,0 +1,96 @@
+version: 2
+project_name: opentelemetry-collector-releases
+env:
+  - COSIGN_YES=true
+  - LD_FLAGS=-s -w
+  - BUILD_FLAGS=-trimpath
+release:
+  replace_existing_artifacts: true
+builds:
+  - id: otelcol-ebpf-profiler-linux
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+    dir: _build
+    binary: otelcol-ebpf-profiler
+    ldflags:
+      - '{{ .Env.LD_FLAGS }}'
+    flags:
+      - '{{ .Env.BUILD_FLAGS }}'
+archives:
+  - id: otelcol-ebpf-profiler
+    builds:
+      - otelcol-ebpf-profiler-linux
+    name_template: '{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}'
```

Mips support can be dropped, I think.
```suggestion
    name_template: '{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ end }}'
```